src/EPPlusTest/ConditionalFormatting/ConditionalFormattingTests.cs

```diff
@@ -42,14 +54,146 @@ namespace EPPlusTest.ConditionalFormatting
     public class ConditionalFormattingTests : TestBase
```

Comment:
A lot of tests here. Might be a good idea to split this class into multiple classes.

Reply:
fair point. I considered it but at the time it was easier to just keep going and the point was partially to make a lot of the tests write to a single easy to overlook file. So those I will most likely make stay but a lot of tests are unthethered to that and mostly about colourscale, databar or Icons. I'll move that into separate classes